csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -447,6 +447,10 @@ Delegation to industry best practices technologies is used in referencing schema
   * Traffic Light Protocol (TLP)
     * Default Definition: https://www.first.org/tlp/
 
+Even though the JSON schema does not prohibit specifically additional properties and custom keywords, it is strongly recommended not to use them. Suggestions for new fields SHOULD be made through issues in the TC's GitHub.
+
+> The standardized fields allow for scalability across different issuing parties and dramatically reduce the human effort and need for dedicated parsers and tools on the side of the consuming parties.
```

Comment:
Nit: `and tools` &rightarrow; `as well as other tools`.
Rationale:
1. Parsers are tools
2. English sentences with two `and`s make me nervous - more interpretations possible like pictures … 😸

Reply:
Done in [d9bd60b](https://github.com/oasis-tcs/csaf/pull/505/commits/d9bd60bbc2125ebae05fa8ada1bc8dcaad22298e)